frontend/src/metabase/nav/components/PathBreadcrumbs/PathBreadcrumbs.tsx

```diff
@@ -0,0 +1,98 @@
+import React, { useState } from "react";
+import PropTypes from "prop-types";
+import cx from "classnames";
+
+import Icon from "metabase/components/Icon";
+import Collection from "metabase/entities/collections";
+import CollectionBadge from "metabase/questions/components/CollectionBadge";
+import { Collection as CollectionType } from "metabase-types/api/collection";
+import { EntitiesState } from "metabase-types/store/entities";
+
+import {
+  PathSeparator,
+  PathContainer,
+  ExpandButton,
+} from "./PathBreadcrumbs.styled";
+
+interface Props {
+  collection: CollectionType;
+  collectionId: string;
+}
+
+const PathBreadcrumbs = ({ collection }: Props) => {
+  const [isExpanded, setIsExpanded] = useState(false);
+  if (!collection) {
+    return null;
+  }
+  const toggle = () => setIsExpanded(!isExpanded);
```

Comment:
opt: there is a [`useToggle` hook](https://github.com/metabase/metabase/blob/master/frontend/src/metabase/hooks/use-toggle.ts) you can use here if you like it :)